docs/source/interaction_model.rst

```diff
@@ -1,95 +1,75 @@
 Interaction model
-================
+================= 
 
-The interaction model defines the user-agent interactions that can take place in a conversation. We define it in terms a set of *user intents*, a set of *agent intents*, and *expected agent responses* to each user intent.
-Below, we specify the YAML format that is used for defining an interaction model and then discuss a specific interaction model (CIR6) that is shipped with the toolkit.
+The interaction model has two main responsibilities: (1) defining the allowed transitions between dialogue acts based on their intents and (2) updating the agenda based on a predefined dialogue strategy.
+
+Define allowed transitions
+--------------------------
+
+The interaction model defines the user-agent interactions in terms of *intents* from their respective dialogue acts. The model specifies a set of *user intents* including required ones, a set of *agent intents*, and *expected agent responses* to each user intent. 
 
 Format
-------
+^^^^^^
+
+Below, we specify the YAML format that is used for defining an interaction model.
 
+* **required_intents**: List of minimum required intents for the user.
 * **user_intents**:  List of all user intent where each should contain at least **expected_agent_intents**.
```

Comment:
"List of all user intent where each should contain at least" => "List of all user intents; each should minimally specify"

Reply:
Done.